src/main/java/com/criteo/nosql/mewpoke/discovery/ConsulDiscovery.java

```diff
@@ -80,7 +80,7 @@ public static String getBucketName(final HealthService.Service service) {
                     .forEach(hsrv -> {
                         logger.debug("{}", hsrv.getNode());
                         nodes.add(new InetSocketAddress(hsrv.getNode().getAddress(), hsrv.getService().getPort()));
-                        srv[0] = new Service(getClusterName(hsrv.getService()), getBucketName(hsrv.getService()));
+                        srv[0] = new Service(getClusterName(hsrv.getService()), getBucketName(hsrv.getService()),null); // we suppose bucket password is null with this discovery
```

Comment:
Add a new constructor to the Service class which does not has a password as an argument instead of passing 'null'